### DISCUSSION TITLE
“Albums” from G Photos (virtual folders) in Aves?

### DISCUSSION THREAD

**Discussion by EnriqueHere** [deckerst/aves#906] [Q&A]
Hello. Can Aves show individual albums (virtual folders, collections without duplicating) created manually by me in Google Photos? These albums show in Google Photos Library under “Albums” - and NOT under “Photos On Device”. I do not see these kind of albums in Aves.

I do NOT mean “albums” in Aves parlance (really folders or directories) for instance for pictures from WhatsApp or Snapseed or other apps that have their own subfolder. These kind of albums\folders show in Google Photos Library under “Photos on Device” - and NOT under “Albums”. They do show in Aves as “albums”, but I don't need them.

So, importantly, Aves and Google Photos mean critically different things by “album”. Aves does show albums in the sense of real folders, but Aves does NOT show albums in the sense of Google photos, meaning virtual photo collections that I created manually, for instance to collect manually my best photos from a certain area in one virtual folder without physically copying or moving the image files.

But in Aves I want to see these virtual folders (albums in the sense of Google photos) that I created manually. Then in Aves I could add tags like “Best of Italy 2024” to all pics within that virtual folder (Google Photos album). That way I could edit\tag pics from my general camera folder and not have duplicates copied into a separate folder.

Thanks

**Comment by deckerst** [comment]
I understand what you mean by Google Photos albums and how they differ from Aves folder albums.

Aves does not connect to your Google account so it cannot know what Google Photos albums you defined and what photos are in them.

This is actually one of the major issues with using Google Photos: everything you do there (albums, captions, locations, etc.) is closed data on Google servers. It is not saved with your photos.

**Reply by EnriqueHere** [reply]
Thibault, thanks for your reply!

But at least with the apps FairEmail and WhatsApp I CAN use pictures from individual Google Photos albums (i.e. virtual folders). And from memory, I think there are more apps that allow me to share individual pictures selected in a Google Photos album, and not in the overall camera folder. 

I just tried that. I think FairEmail only shows the 3 most recently used albums. I think WhatsApp offers all albums via a rather complicated tap-procedure - but then all Google Photos albums were offered.

Also, generally, screenshots that live in their own Screenshots folder are always offered together with the photos when I want to share something.

I have no idea, but I wondered if the Google Photos albums exist as link lists available to other apps. Anyway they are approachable from other apps, it's not completely closed data.
